taskcluster/ci/build/kind.yml

```diff
@@ -11,26 +11,64 @@ transforms:
     - taskgraph.transforms.task:transforms
 
 job-defaults:
+    artifact-template:
+        type: file
+        name: public/build/{artifact_file_name}
+        path: '/builds/worker/checkouts/src/{component_path}/build/maven/org/mozilla/components/{component}/{version_with_snapshot}/{artifact_file_name}'
+    expose-artifacts:
+        by-build-type:
+            release: true
+            snapshot: true
+            default: false
     attributes:
-        code-review: true
+        code-review:
+            by-build-type:
+                release: false
+                snapshot: false
+                default: true
     description: Execute Gradle tasks for component "{component}"
     # XXX Cannot use run.gradlew because the loader concatenates lists
-    include-default-gradle-commands: true
-    include-coverage: true
-    lint-release: true
+    default-gradle-commands:
+        by-build-type:
+            release:
+                - ':{component}:assembleRelease'
+                - ':{component}:testRelease'
+                - ':{component}:lintRelease'
+                - ':{component}:publish'
+            snapshot:
+                - '-Psnapshot'
+                - '-Ptimestamp={timestamp}'
+                - ':{component}:assemble'
+                - ':{component}:test'
+                - ':{component}:lint'
+                - ':{component}:publish'
+                - ':{component}:renameSnapshotArtifacts'
+            default:
+                - ':{component}:assemble'
+                - ':{component}:assembleAndroidTest'
+                - ':{component}:test'
+                - ':{component}:lintRelease'
+    use-default-gradle-commands: true
```

Comment:
Instead of `default-gradle-commands`, can you set `gradlew` here, and have the overrides below override it, rather than having `use-default-gradlew-commands` to control setting `gradlew`?

Reply:
Removed per https://github.com/mozilla-mobile/android-components/pull/4646#discussion_r331975368